README.md

```diff
@@ -164,7 +164,8 @@ ActionController::Parameters.action_on_invalid_parameters = :log
 |--------------------------------|-------------------------------------------------------------------------|
 | Parameters.string              | value.is_a?(String)                                                     |
 | Parameters.integer             | value.is_a?(Fixnum) or '-1'                                             |
-| Parameters.float               | value.is_a?(Float)  or '-1.2'                                           |
+| Parameters.float               | value.is_a?(Float) or '-1.2'                                            |
+| Parameters.datetime            | value.is_a?(DateTime) or '2014-05-13'                                   |
```

Comment:
Not sure if you want this as a replacement, or as an addition to the date version. I think the latter makes more sense, so went with that.